src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,330 @@
+* PayID client: the endpoint that initiates PayID protocol/sending side of the transaction.
+* PayID server: the endpoint that returns payment account(s) information/receiving side of the transaction (custodial or non-custodial wallets, exchanges, etc).
+* receiver: individual or entity receiving the transaction.
+* sender: individual or entity originating the transaction/owner of the PayID[PayID-URI][].
+* wallet: the host of the funds of the `sender`; may or may not be custodied.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol designed to provide a set of standard APIs and flows, 
+which can be used to transact money between two entities in a way that requires:
+* minimal effort for the user initiating the transaction.
+* no server-side software specific to PayID or its protocols for servicing the transaction.
+* only UI-based solutions.
+
+## Motivation
+
+The PayID Easy Checkout Protocol aims to correct the current absence of a consistent and broadly adopted pattern for 
+paying for goods and services via a digital wallet on an e-commerce website.
+Given the ability to assign arbitrary metadata to a PayID as defined in [PayID-Discovery][], there is an opportunity
+to standardize the set of interactions between receiver and sender, specifically the process by which a receiver
+directs a sender to their digital wallet to complete a payment.
+We believe this protocol will enable a largely improved user experience in e-commerce transactions by reducing the number
+of steps a user must take to complete a transaction, creating a consistent and familiar checkout pattern, and lowering
+the barrier to entry use for cryptocurrency novices.
+
+The second priority of PayID Easy Checkout is to limit the engineering effort needed to implement the protocol. 
+Clients wishing to adopt this pattern should only need to implement UI-level changes in order to make the flow function 
+as intended, which may aid in expanding overall adoption, further enhancing the protocol's user experience benefits. 
+
+## Design Goals
+
+### Minimal effort for the user initiating the transaction
+
+The PayID Easy Checkout protocol requires a small number of points of data from the user:
+* Their PayID
+* Identifying themselves with the wallet they end up redirected to in order to approve transfer of funds to the receiver.
+
+### No server-side software specific to PayID or its protocols for servicing the transaction
+
+Since the flow of PayID Easy Checkout is predicated on using the PayID Discovery Protocol and then redirecting the 
+sender away from the site of the receiver, all of the flow can be instrumented in the browser and doesn't require server-side resources. 
```

Comment:
Is EZ Checkout browser only?

Reply:
We should think about this.  A wallet could deeplink to their app, but that would only work on a mobile browser/app.  I guess an app-only wallet could do something like Xumm and show a QR code and listen for the transaction, redirecting back after it has been completed on the phone.  

Seems like it's out of scope for this spec